docs/understanding-the-flutterflow-ui/projects/how-to-create-find-organize-projects.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: how-to-create-find-organize-projects
+title: How to Create, Find, and Organize Projects
+tags: []
+toc_max_heading_level: 5
+---
+# How to Create a Project
```

Comment:
@johnbhiggins7 do you think we should be including screenshots or recording arcade's for these? I feel like we need something visual here

Reply:
@leighajarett totally. I was adding all the text first and then going to add in the visuals next.